src/windows/users.rs

```diff
@@ -139,8 +139,10 @@ unsafe fn get_groups_for_user(username: PCWSTR) -> Vec<Group> {
         if !buf.0.is_null() {
             let entries = std::slice::from_raw_parts(buf.0, nb_entries as _);
             groups.extend(entries.iter().map(|entry| Group {
-                name: to_str(entry.lgrui0_name),
-                id: Gid(0),
+                inner: GroupInner {
+                    gid: Gid(0),
```

Comment:
That's the main reason why I didn't provide a `Group` API: Windows `gid` is pointless, making the API kinda useless and incoherent on this platform.